Sources/PackageLoading/ManifestJSONParser.swift

```diff
@@ -675,6 +675,23 @@ extension TargetBuildSettingDescription.Kind {
                 throw InternalError("invalid (empty) build settings value")
             }
             return .linkedFramework(value)
+        case "interoperabilityMode":
+            guard let rawLang = values.first else {
+                throw InternalError("invalid (empty) build settings value")
+            }
+            guard let lang = TargetBuildSettingDescription.InteroperabilityMode(rawValue: rawLang) else {
+                throw InternalError("unknown interoperability mode")
```

Comment:
I personally would prefer the cause of the error to be surfaced in the error text for easier troubleshooting.
```suggestion
                throw InternalError("unknown interoperability mode: \(rawLang)")
```

Reply:
That makes sense, thanks!